app/containers/SearchMode/index.js

```diff
@@ -1,36 +1,51 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 
 import Label from 'components/Label';
 import Button from 'components/Button';
+import { setSearchFilterMode } from '../App/actions';
+import { searchBy } from '../../stateValues';
 
-export default class SearchMode extends React.Component { // eslint-disable-line react/prefer-stateless-function
-    constructor(props) {
-        super(props); 
-        let mode = props.value == undefined || props.value == true ? true: false;
-        this.state = {
-            titleMode: mode,
-            focusColor: 'button-background-color'
-        };
-    }
+class SearchMode extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  constructor(props) {
+    super(props);
+    this.focusColor = 'button-background-color';
```

Comment:
move focusColor to public property (no need to assign it in constructor)

Reply:
Done